acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/BesuNodeRunner.java

```diff
@@ -54,4 +54,19 @@ default void waitForPortsFile(final Path dataDir) {
               }
             });
   }
+
+  default void waitForFile(final Path dataDir, final String fileName) {
+    final File file = new File(dataDir.toFile(), fileName);
+    Awaitility.waitAtMost(30, TimeUnit.SECONDS)
+        .until(
+            () -> {
+              if (file.exists()) {
+                try (final Stream<String> s = Files.lines(file.toPath())) {
+                  return s.count() > 0;
+                }
+              } else {
+                return false;
+              }
```

Comment:
`lines` will throw an a `FileNotFoundException` anyways so I believe you can condense this into:
```
              try (final Stream<String> s = Files.lines(file.toPath())) {
                return s.count() > 0;
              } catch (FileNotFoundException fnfe) {
                return false;
              }
```

Reply:
Done